src/components/Channel/hooks/index.ts

```diff
@@ -0,0 +1,2 @@
+export { useEditMessageHandler as useChannelEditMessageHandler } from './useEditMessageHandler';
+export { useMentionsHandlers as useChannelMentionsHandler } from './useMentionsHandlers';
```

Comment:
why not just rename our files and exports to this in general then?

Reply:
I believe it's to differentiate between the edit and mentions handlers that are exported from Message